### ISSUE TITLE
logger should be usable using only keyboard

### ISSUE THREAD

**Issue by msphair** [loftTech/obsidian-oblogger#105]

**Comment by msphair** [comment]
this should be broken down into specific keyboard features that are missing. my current guess is: - tab ordering
- context-specific enter/escape handling (getting out of popups, selecting items from popups)

but I'm sure there's more

**Comment by robotbrad** [comment]
tab index as a search term
